src/sqlfluff/dialects/dialect_snowflake.py

```diff
@@ -3134,7 +3174,10 @@ class CsvFileFormatTypeParameters(BaseSegment):
         Sequence(
             "NULL_IF",
             Ref("EqualsSegment"),
-            Bracketed(Delimited(Ref("QuotedLiteralSegment"))),
+            OneOf(
+                Bracketed(),
+                Bracketed(Delimited(Ref("QuotedLiteralSegment")))
+            ),
```

Comment:
Would this have worked instead?
```suggestion
            Bracketed(Delimited(Ref("QuotedLiteralSegment")), optional=True),
```

Reply:
Actually, yes, it works but with small changes: 
`Bracketed(Delimited(Ref("QuotedLiteralSegment"), optional=True))`
I'll make these changes  now